<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalRequest</class>
 <widget class="QDialog" name="AdditionalRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Consultation</string>
  </property>
  <widget class="QLabel" name="ValueLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValueEditor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonFinish">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save And Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveAndContinue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Finish Consultation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FactSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="FactLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Fact:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ConfidenceEditor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="ConfidenceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confidence Factor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Are you able to supply a value for any of these Facts?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FactSelector</tabstop>
  <tabstop>ValueEditor</tabstop>
  <tabstop>ConfidenceEditor</tabstop>
  <tabstop>ButtonFinish</tabstop>
  <tabstop>ButtonSaveAndContinue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>FactSelector</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>AdditionalRequest</receiver>
   <slot>FactSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonFinish</sender>
   <signal>clicked()</signal>
   <receiver>AdditionalRequest</receiver>
   <slot>SaveAndContinue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonSaveAndContinue</sender>
   <signal>clicked()</signal>
   <receiver>AdditionalRequest</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FactSelected()</slot>
  <slot>SaveAndContinue()</slot>
 </slots>
</ui>
